mkdocs.yml

```diff
@@ -105,6 +105,7 @@ nav:
           - Locations: 'integrations/github/locations.md'
           - Discovery: 'integrations/github/discovery.md'
           - Org Data: 'integrations/github/org.md'
+          - GitHub Apps for Backend Authentication: 'integrations/github/github-apps.md'
```

Comment:
isn't this a bit long :)

Reply:
It's insanely long. Could you `suggest` a better name? GitHub Apps? Github Apps for Auth?

Reply:
Just `GitHub Apps` and mention the auth bits there as well? :grin: